src/components/status/status.ts

```diff
@@ -0,0 +1,76 @@
+import { CSSResultGroup, html, LitElement, nothing, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+import { setAttribute } from '../core/dom';
+import type { TitleLevel } from '../title';
+
+import style from './status.scss?lit&inline';
+
+import '../icon';
+import '../title';
+
+const statusTypes = new Map([
+  ['info', 'circle-information-small'],
+  ['success', 'circle-tick-small'],
+  ['warning', 'circle-exclamation-point-small'],
+  ['error', 'circle-cross-small'],
+]);
+
+/**
+ * Displays a message to the user's attention.
+ *
+ * @slot - Use the unnamed slot to add content to the status message.
+ * @slot title - Use this to provide a title for the status (optional).
+ */
+@customElement('sbb-status')
+export class SbbStatus extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  /**
+   * The type of the status.
+   */
```

Comment:
Inline comments where possible, same for other comments below.

```suggestion
  /** The type of the status. */
```